engines/v8/args.gn

```diff
@@ -0,0 +1,9 @@
+is_component_build = false
+use_custom_libcxx = false
+v8_enable_fast_mksnapshot = true
+v8_enable_i18n_support = false
+v8_use_external_startup_data = false
+is_debug = true
+symbol_level = 2
+v8_optimized_debug = false
```

Comment:
Does this mean we are building V8 with debug symbols and/or without optimizations?